### PR TITLE
update @cumulus/common and @cumulus/test-data readme and package.json

### PR DESCRIPTION
**Pull request by sethvincent**

### CODE REVIEW

**Comment by sethvincent** [comment]
In circle ci this is happening:

```
lerna ERR! error An unexpected error occurred: "https://registry.yarnpkg.com/@cumulus%2fcommon: Not found".
```

It seems like the reason the test is failing, but I'm not sure.

File: packages/test-data/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cumulus/test-data",
-  "version": "1.0.0-beta.581d31f4",
+  "version": "1.0.0-beta.111",
   "description": "Includes the test data for various packages",
   "main": "index.js",
```
Comment (abarciauskas-bgse) [inline packages/test-data/package.json]: Should this line be removed?

Comment (sethvincent) [reply packages/test-data/package.json]: 👍 yes

**Comment by sethvincent** [comment]
OK, I figured this out while I was waiting for my delayed plane ✈️.

When I bumped the version number for testing out npm publishing, I checked in those new versions. There were version mismatches all over the place, so when yarn went to install @cumulus/common or @cumulus/test-data it didn't find the right version locally and tried to download it, and failed.

I've switched it back so that these two packages have the version number from before test publishing

File: packages/common/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0-beta.581d31f4",
   "description": "Common utilities used across tasks",
   "keywords": [
-    "GIBS"
+    "cumulus"
```
Comment (yjpa7145) [inline packages/common/package.json]: This is different than the keywords defined in the top-level package.json.  Not sure if that matters.

```
"keywords": [
    "GIBS",
    "CUMULUS",
    "NASA"
  ]
```

**Pull request merged**